src/styles/style.css

```diff
@@ -0,0 +1,132 @@
+@import './utils/reset.css';
+
+.card {
+  border: 1px solid #f3f3f3;
+  width: 200px;
+  padding: 32px 16px 16px;
+}
+
+.card__description {
+  color: #060b35;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  margin-bottom: 4px;
+}
+
+.card__code {
+  color: #616070;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card__reviews {
+  color: #060b35;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card__stars {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 24px;
+  margin-top: 16px;
+}
+
+.card__img {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.card__cost {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+}
+
+.card__price {
+  color: #616070;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.card__total {
+  color: #060b35;
+  line-height: 18px;
+  font-weight: 700;
+}
+
+.card__button {
+  height: 40px;
+  width: 100%;
+  border-radius: 5px;
+  font-size: 14px;
+  background-color: #00acdc;
+  color: #fff;
+  box-sizing: border-box;
+  border: 1px solid #00acdc;
+}
+
+.card__button:hover {
+  color: #00acdc;
+  background-color: #fff;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-left: 4px;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-image: url('../images/star.svg');
+}
+
+.stars__star:first-child {
+  margin-left: 0;
+}
+
+.stars--0 .stars__star:nth-child(-n + 0) {
```

Comment:
The selector `.stars--0 .stars__star:nth-child(-n + 0)` does not select any elements because `nth-child(-n + 0)` is not valid. Consider removing this rule or adjusting the logic.